doc/source/overview/legacy_kiwi.rst

```diff
@@ -109,6 +109,17 @@ consider to use the legacy KIWI version.
    Many distributions also dropped the lxc tools from the distribution
    in favour of docker.
 
+*  OEM Recovery/Restore
+
+   Recovery/Restore in the world of images has been moved from the OS
```

Comment:
This sentence is not really "wrong", but I would suggest to be more precise here:

"Recovery and restore in the world of images has been moved from the operating system..."